### PR TITLE
Translate tutorials page [partial/wip]

### PR DESCRIPTION
**Pull request by ncclementi**
This is the beginning of the translation of the tutorials page. 

So far there is a partial translation of the `tutorials/add-license-coc.html` page. 

xref: https://github.com/pyOpenSci/python-package-guide/issues/313

### CODE REVIEW

File: locales/es/LC_MESSAGES/tutorials.po [hunk 1]
```diff
@@ -22,94 +22,107 @@ msgstr ""
...
 msgid "What is a license?"
-msgstr ""
+msgstr "¿Qué es una licencia?"
 
 #: ../../tutorials/add-license-coc.md:21
 msgid ""
 "A license contains legal language about how users can use and reuse your "
 "software. To set the `LICENSE` for your project, you:"
-msgstr ""
+msgstr "Una licencia contiene lenguage legal acerca de como los usuarios pueden "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/tutorials.po:81]:
```suggestion
msgstr "Una licencia contiene lenguaje legal acerca de como los usuarios pueden "
```

File: locales/es/LC_MESSAGES/tutorials.po [hunk 2]
```diff
@@ -22,94 +22,107 @@ msgstr ""
...
+msgstr "Creaste un archivo `README.md` para tu paquete the Python científico"
 
 #: ../../tutorials/add-license-coc.md:7
 msgid ""
 "<i class=\"fa-solid fa-circle-check\" style=\"color: #703c87;\"></i> "
 "Learned about the core components that are useful to have in a `README` "
 "file."
-msgstr ""
+msgstr "Aprendiste acerca de los componentes principales que son útiles de tener en "
+"un `README` file."
```
Comment (flpm) [inline locales/es/LC_MESSAGES/tutorials.po:43]:
```suggestion
"un archivo `README`."
```

File: locales/es/LC_MESSAGES/tutorials.po [hunk 3]
```diff
@@ -22,94 +22,107 @@ msgstr ""
...
 #: ../../tutorials/add-license-coc.md:26
 msgid ""
 "By adding the `LICENSE` file to your `pyproject.toml` file, the `LICENSE`"
 " will be included in your package's metadata which is used to populate "
 "your package's PyPI landing page. The `LICENSE` is also used in your "
 "GitHub repository's landing page interface."
-msgstr ""
+msgstr "Al incluir el archivo `LICENSE` en tu archivo `pyproject.toml`, la "
+"`LICENSE` sera incluída en la metadata de tu paquete y esta es utilizada para"
+" rellenar la página de entrada de tu paquete en PyPI. La `LICENSE` tambien es "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/tutorials.po:105]:
```suggestion
" rellenar la página de entrada de tu paquete en PyPI. La `LICENSE` también es "
```

File: locales/es/LC_MESSAGES/tutorials.po [hunk 4]
```diff
@@ -22,94 +22,107 @@ msgstr ""
...
+" utilizada en la página de entrada de tu repositorio de GitHub"
 
 #: ../../tutorials/add-license-coc.md:28
 msgid "What license should you use?"
-msgstr ""
+msgstr "¿Qué licencia debería elegir?"
```
Comment (flpm) [inline locales/es/LC_MESSAGES/tutorials.po:110]:
```suggestion
msgstr "¿Qué licencia deberías elegir?"
```

File: locales/es/LC_MESSAGES/tutorials.po [hunk 5]
```diff
@@ -22,94 +22,107 @@ msgstr ""
...
 
 #: ../../tutorials/add-license-coc.md:30
 msgid ""
 "We suggest that you use a permissive license that accommodates the other "
 "most commonly used licenses in the scientific Python ecosystem (MIT[^mit]"
 " and BSD-3[^bsd3]). If you are unsure, use MIT given it's the generally "
 "recommended license on [choosealicense.com](https://choosealicense.com/)."
-msgstr ""
+msgstr "Nosotros sugerimos que uses una licencia permissiva que acomode otras"
+" licencias comunmente usadas en el ecosistema de Python científico (MIT[^mit]"
```
Comment (flpm) [inline locales/es/LC_MESSAGES/tutorials.po:119]:
```suggestion
" licencias comúnmente usadas en el ecosistema de Python científico (MIT[^mit]"
```

**Review by flpm — COMMENTED** [review]
Thanks Naty for helping with the translation!

I made a few minor change suggestions.

Also, looks like there is an issue with the CI unrelated with the translation, because a link in the guide has changed, I made [PR#383](https://github.com/pyOpenSci/python-package-guide/pull/383) to fix the link, so when that is merged there should be no error.

**Comment by flpm** [comment]
There is an issue with the footnotes at /tutorials/add-license-coc.md:30:

```
WARNING: inconsistent footnote references in translated message. original: ['[^mit]', '[^bsd3]'], translated: []
```

I will merge the PR and create an issue to track, since I don't think is related to the translation

**Comment by flpm** [comment]
Thanks again for helping with the translation!

**Comment by tkoyama010** [comment]
@all-contributors please add @ncclementi for translation

**Pull request merged**